libkbfs/keybase_service_base.go

```diff
@@ -425,7 +425,15 @@ func (k *KeybaseServiceBase) LoadUserPlusKeys(ctx context.Context,
 	uid keybase1.UID, pollForKID keybase1.KID) (UserInfo, error) {
 	cachedUserInfo := k.getCachedUserInfo(uid)
 	if cachedUserInfo.Name != libkb.NormalizedUsername("") {
-		return cachedUserInfo, nil
+		if pollForKID == keybase1.KID("") {
+			return cachedUserInfo, nil
+		}
+		// Skip the cache if pollForKID isn't present.
```

Comment:
Yes, that's the way it was before this PR.  Now, with this PR, we're busting the local cache here, even in `HasVerifyingKey`.